articles/constants-and-variables.html

```diff
@@ -978,7 +998,7 @@ <h3>Scopes Of Variables And Named Constants</h3>
 		// In this code block, the just new declared x and y
 		// shadow the outer declared same-name identifiers.
 		x, z := !x, y/10 // only z is new declared.
-		y /= 100
+		y /= 100 // inner block variable not the package level constant

```

Comment:
Generally, contiguous comment lines  should be aligned, except they are hard to comply to this rule. So here I suggest change the y line as
```
		x, z := !x, y/10 // only z is new declared.
		y /= 100         // the y is the inner variable y.
```

I also think this new comment is not very essential here, for above lines have explained it.

Reply:
I hesitated to add the comment, but then it seemed fine.
But I remove it.